var/spack/repos/builtin/packages/tensorflow/package.py

```diff
@@ -0,0 +1,291 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from glob import glob
+
+
+class Tensorflow(Package):
+    """TensorFlow is an Open Source Software Library for Machine Intelligence
+    """
+
+    homepage = "https://www.tensorflow.org"
+    url      = "https://github.com/tensorflow/tensorflow/archive/v0.10.0.tar.gz"
+
+    version('2.0.0',        sha256='49b5f0495cd681cbcb5296a4476853d4aea19a43bdd9f179c928a977308a0617')
+    version('1.15.0',       sha256='a5d49c00a175a61da7431a9b289747d62339be9cf37600330ad63b611f7f5dc9')
+    version('1.14.0',       sha256='aa2a6a1daafa3af66807cfe0bc77bfe1144a9a53df9a96bab52e3e575b3047ed')
+    version('1.13.2',       sha256='abe3bf0c47845a628b7df4c57646f41a10ee70f914f1b018a5c761be75e1f1a9')
+    version('1.13.1',       sha256='7cd19978e6bc7edc2c847bce19f95515a742b34ea5e28e4389dade35348f58ed', preferred=True)
+    version('1.12.3',       sha256='b9e5488e84f4a133ed20b18605f0cd6301f11d356bd959712db4e7b9301d0462')
+    version('1.12.0',       sha256='3c87b81e37d4ed7f3da6200474fa5e656ffd20d8811068572f43610cae97ca92')
+    version('1.9.0',        sha256='ffc3151b06823d57b4a408261ba8efe53601563dfe93af0866751d4f6ca5068c')
+    version('1.8.0',        sha256='47646952590fd213b747247e6870d89bb4a368a95ae3561513d6c76e44f92a75')
+    version('1.6.0',        sha256='03cf1423446abbead6bd8c3cf6e6affa7d99746cd119691b012aac9a1795f4fb')
+    version('1.5.0',        sha256='0642781c3a3a8c2c4834b91b86aec385f0b2ada7d721571458079478cc5b29c8')
+    version('1.3.0',        sha256='e1af1bb767b57c3416de0d43a5f74d174c42b85231dffd36f3630173534d4307')
+    version('1.2.0',        sha256='03dbf7548d1fc1c11ed58da5fa68616f795c819f868f43478cbcaa26abed374f')
+    version('1.1.0',        sha256='aad4470f52fa59f54de7b9a2da727429e6755d91d756f245f952698c42a60027')
+    version('1.0.0-rc2',    sha256='bf5c58c59c182d0bdf07018f9fd394f7b760341bb432e3602874928929cf8495')
+    version('0.10.0',       sha256='f32df04e8f7186aaf6723fc5396733b2f6c2fd6fe4a53a54a68b80f3ec855680')
+
+    variant('gcp', default=False,
+            description='Enable Google Cloud Platform Support')
+
+    variant('cuda', default=False,
+            description='Enable CUDA Support')
+
+    variant('nccl', default=False,
+            description='Enable NCCL Support')
```

Comment:
Might want this to be True too. Our `py-torch` package also defaults to `+nccl`.